### DISCUSSION TITLE
某app会把v2rayNG识别为入侵工具

### DISCUSSION THREAD

**Discussion by Ca1n73** [2dust/v2rayNG#3689] [General]
app名是limbus company，请问一下作者可以修复吗？![S41013-08171873_com.ProjectMoon.LimbusCompany(1).png](https://github.com/user-attachments/assets/f1bc5bdb-a7a2-443c-ba2b-c8c603784146)

使用其他软件则不会出现该问题，比如nekobox之类的。

v2rayNG:1.9.1

**Comment by rocker18** [comment]
我也遇到这个问题

**Comment by ShadowScaleFTL** [comment]
I have same problem! I can't run limbus company with v2rayng installed! Pls help!

**Comment by 2dust** [comment]
不知道这个 app 是如何判定的，无法解决的。
